Bottom: 9260799154fc097a2f0bb868a51297ba36ac4816
Top:    8b24319740a95873bd89bbaf966befd6f92dc27c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:41:46 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 43055b2..7c01b55 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1486,9 +1486,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
